src/Config/ConfigUpdateResult.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Drupal\commerce\Config;
+
+/**
+ * Result object for displaying configuration update results.
+ */
+class ConfigUpdateResult {
+
+  /**
+   * The configuration names that succeeded to update.
+   *
+   * @var \Drupal\Component\Render\MarkupInterface[]
```

Comment:
I consider MarkupInterface to be an internal Drupal implementation detail, it has a __toString() method, we should treat it as a string.